src/components/dashboard/ConsentSteps.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useEffect, useState } from 'react'
+
+import { ReactComponent as LogoNoText } from '../../assets/logo-no-text.svg'
+import { ReactComponent as ArrowButtonLeft } from '../../assets/arrow_button_left.svg'
+import { ReactComponent as ArrowButtonRight } from '../../assets/arrow_button_right.svg'
+import Button from '@material-ui/core/Button/Button'
+import moment from 'moment'
+import i18next from 'i18next'
+
+import SageForm from '../form/SageForm'
+import { COUNTRIES, PARTICIPATE_OPTIONS } from '../form/types'
+import ProgressBar from '../progressBar/ProgressBar'
+
+const MAX_STEPS: number = 4
+
+const ConsentSteps: React.FunctionComponent = () => {
+  const [step, setStep] = useState(1)
+  const [consented, setConsented] = useState(false)
+  const [signatureName, setSignatureName] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+
+  useEffect(() => {
+    setErrorMessage('')
+    setSignatureName('')
+    setConsented(false)
+  }, [step])
+
+  if (step === MAX_STEPS) {
+    return (
+      <div className="textStepWrapper">
+        <ProgressBar step={step} maxSteps={MAX_STEPS} />
+        <LogoNoText />
+        <div className="headerWrapper">
+          <h1>Consent Signature</h1>
+        </div>
+        <h3>
+          Please <b>check the box below</b> if you <b>agree</b> to take part:
+        </h3>
+        <span className="consentWrapper">
+          <input
+            type="checkbox"
+            id="consented"
+            value="consented"
+            onClick={() => setConsented(prev => !prev)}
+          />
+          <div>
+            <b>
+              I have read and understand the information presented in this
+              consent process. All of my questions have been answered. I freely
+              and willingly choose to participate in the MindKind study. By
+              signing this consent, I have not given up any of my legal rights.
+            </b>
+          </div>
+        </span>
+
+        <h2>{moment().locale(i18next.language).format('MMMM Do, YYYY')}</h2>
+
+        <fieldset>
+          <legend>Full Name of adult participant:</legend>
+          <input
+            type="text"
+            value={signatureName}
+            onChange={e => {
+              setSignatureName(e.target.value)
+            }}
+          ></input>
+        </fieldset>
+
+        <Button
+          type="button"
+          variant="contained"
+          fullWidth
+          color="primary"
+          style={{ margin: '40px 0' }}
+          //onClick={() => alert('sent')}
```

Comment:
can we remove commented code :)

Reply:
Yes, sure. There are still some things to clean up throughout the project, I'll add those ones to this PR.